src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
As an improvement, BASE_URL can be extracted to .env as it's not recommended to store some private data in the repository (in this case it's public, but you should remember about it)